src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5849,7 +5849,7 @@ public void BinaryFileErrorTest()
             CleanupAllGeneratedFiles(binaryPath);
         }
 
-#if !NETCOREAPP2_1
+#if !NETCOREAPP3_1
```

Comment:
💡 You can use `NETCOREAPP` if you don't want to keep changing these in the future

Reply:
Hmm I'd feel more comfortable with that if I can managee to remove our `netcoreapp1.1` dependency. Will try that in a follow up PR